### ISSUE TITLE
Manager left menu charset issue

### ISSUE THREAD

**Issue by Taffa** [evolution-cms/evolution#692]
In manager left menu the utf8 characters are not showing up correctly. This applies both to UI labels (hover over the nodes) and page names. Looks like ajax.php is returning content as ISO 8859-1 for some reason.
On the right hand section both UI and page content looks ok.

Language file is utf8 and $modx_manager_charset = 'utf-8'
Also settings is utf-8 and htaccess sets utf-8 as default
